### DISCUSSION TITLE
How to inference with onnx model

### DISCUSSION THREAD

**Discussion by jiamfeng** [open-edge-platform/anomalib#1591] [Q&A]
### What is the motivation for this task?

I have successed review the code, and export model to onnx, now I want to inference this model in onnx type

### Describe the solution you'd like

Does have the command to inference the model with onnx type

**Comment by willyfh** [comment]
Hello, you can use the OpenVINO inference script for making inferences with an ONNX model. The script can be found here: https://github.com/openvinotoolkit/anomalib/blob/main/tools/inference/openvino_inference.py
